packages/unenv-preset/src/preset.ts

```diff
@@ -122,7 +125,9 @@ export function getCloudflarePreset({
 			clearImmediate: false,
 			setImmediate: false,
 			console: "@cloudflare/unenv-preset/node/console",
-			process: "@cloudflare/unenv-preset/node/process",
+			process: processOverrides.isV2
+				? false
+				: "@cloudflare/unenv-preset/node/process",
```

Comment:
The approach I took with `console` [here](https://github.com/cloudflare/workers-sdk/pull/10606/files#diff-654c3da8b90fcbed9f2ed91a3a8d2954093e97dc0ca5e2fc3facb2b1fd6209bcR127)
was to return an `inject` property  to use here, which feels cleaner and more similar to how we expose the native vs polyfilled modules.

Reply:
Great idea, I will update!